### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by IvanovVadym**

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -13,340 +19,24 @@ else if(process.env.NODE_ENV==="production")
   server_url="https://hored.azurewebsites.net"
 
 class App extends Component {
-  render() {
+render() {
     return (
         <div className="App container-fluid">
```
Comment (ira-halenok) [inline frontend/src/App.js]: We should delete that classes (unneccesary margin is getting)

File: frontend/src/components/AdminRulesPage.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                <div>
+                    <RulesList/>
+                </div>
+            </div>
+        );
+    }
+}
+
+var server_url;
+/*if(process.env.NODE_ENV==="development")
```
Comment (ira-halenok) [inline frontend/src/components/AdminRulesPage.js]: @olegRakus did you forget to uncomment ?

File: frontend/src/components/Authorization.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    constructor(props){
+      super(props);
+      this.loginAuth='';
+      this.passwordAuth='';
+    }
+    
+    handleSubmitAuth = event => 
+    {
+      event.preventDefault();
+      //const
```
Comment (ira-halenok) [inline frontend/src/components/Authorization.js]: I guess unneccesary comment

File: frontend/src/components/DoctorsListWithSomeRule.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Component } from 'react';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
+
+var server_url;
+/*if(process.env.NODE_ENV==="development")
+  server_url="http://localhost:58511"
+else if(process.env.NODE_ENV==="production")*/
+  server_url="https://hored.azurewebsites.net"
```
Comment (ira-halenok) [inline frontend/src/components/DoctorsListWithSomeRule.js]: @olegRakus one more time

File: frontend/src/components/AdminRulesPage.js (hunk 2 above)
Comment (olegRakus) [reply frontend/src/components/AdminRulesPage.js]: @ira-halenok yes, sorry

**Pull request merged**